test/test_playbooks/tasks/job_template_from_file.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: "Ensure job template from file '{{ job_template_file_name }}' is '{{ job_template_state }}'"
+  vars:
+    job_template_name: "configure flux compensator"
+    job_template_provider_type: "SSH"
```

Comment:
If you add just the ORG/LOC parameter here, you can safely "not omit them" in the inline test case.